setup.cfg

```diff
@@ -48,4 +48,5 @@ asyncio_mode = strict
 addopts =
     --tb=short
     --doctest-modules
+    --cov=pytest_pyodide --cov-report xml
```

Comment:
This should be in the CI setup if needed IMO. Locally there is no need to display the coverage each time.